src/pages/MyPage/MyPage.js

```diff
@@ -1,12 +1,55 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Icon from '@components/Icon';
 import ProfileEdit from '@components/ProfileEdit';
+import { getUserApi } from '@repository/loginRepository';
 import S from './MyPage.style';
 import Box from './Box';
 
 export default function MyPage() {
   const name = sessionStorage.getItem('name');
   const email = sessionStorage.getItem('email');
+  const [info, setInfo] = useState([]);
+  const fetch = async () => {
+    getUserApi().then((response) => {
+      const data = response.data;
```

Comment:
이부분
```suggestion
      const { data } = response;
```
이렇게도 쓸 수 있을 것 같아요!

Reply:
아 그러네요! 감사합니다